.github/workflows/doc-publish.yml

```diff
@@ -2,18 +2,16 @@
 # Copyright (c) 2021 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-name: Publish Documentation
+name: Documentation Publish
 
 on:
   workflow_run:
     workflows: ["Documentation Build"]
     branches:
-      - main
-      - v*
-    tags:
```

Comment:
why are we removing tags?

Reply:
because `tags` is not valid for `workflow_run`.